spec/text_normalizer/normalize_title_spec.rb

```diff
@@ -39,5 +43,10 @@
       title = 'b' * (MetaTags.config.title_limit + 10)
       expect(subject.normalize_title(site_title, title, '-')).to eq("#{site_title}-#{'b' * (MetaTags.config.title_limit - 21)}")
     end
+
+    it 'should escape ampersand in title correctly' do
+      site_title = 'a' * 10
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.